### PR TITLE
Propagate tags from the targets to the actions - approved

### PR DESCRIPTION
**Pull request by ishikhman**
There has been no comments since almost 2 weeks + LGTM from @laurentlb , therefore I assume that proposal could be moved to 'approved' state.

Link to the doc: https://docs.google.com/document/d/1X2GtuuNT6UqYYOK5lJWQEdPjAgsbdB3nFjjmjso-XHo/edit?usp=sharing

### CODE REVIEW

**Comment by laurentlb** [comment]
cc @c-parsons, any comments?

**Comment by c-parsons** [comment]
Design LGTM
(Minor comment that lets explicitly make sure to update the documentation to list the new whitelisted tags)

**Comment by ishikhman** [comment]
Added an action item regarding documentation update to the proposal.

**Comment by laurentlb** [comment]
LGTM.

I think @buchgr should also explicitly LGTM it.

**Comment by buchgr** [comment]
LGTM. I added one comment to the document. I don't think the "local" tag should be made an execution requirement (more details in the doc)

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ list.
 
 | Last updated | Title                                                                                                                  | Author(s) alias                                                                    | Category              |
 | ------------ | ------------------------------------------------------------------------------------------------------------------     | ---------------------------------------------------------------------------------- | --------              |
+|   2019-04-12 | [Propagate tags from the targets to the actions](https://docs.google.com/document/d/1X2GtuuNT6UqYYOK5lJWQEdPjAgsbdB3nFjjmjso-XHo/edit?usp=sharing)| [@ishikhman](https://github.com/ishikhman)              | Bazel, Starlark       |
```
Comment (buchgr) [inline README.md]: s/to the actions/to actions/

Comment (ishikhman) [reply README.md]: done

**Pull request merged**